### ISSUE TITLE
Edit entire blog post

### ISSUE THREAD

**Issue by sdmccloskey** [danielmahon/gatsby-starter-procyon#5]
Is there a way to have the logged in user be able to edit the title and image along with the actual post? All I see is the ability to edit the main content.

**Comment by danielmahon** [comment]
@sdmccloskey yes, as is you would have to use the `EditableMarkdown` component where the title is and add another mutation or edit the existing one so that text put there would update the title instead of the body content. Take a look at the code in `gatsby-starter-procyon/src/templates/post.js`. Should give you an idea of how its done for the content. You just need to implement it for any other pieces of content you want the live editor to be enabled for. Might be worth making a `EditableString` component to handle string only fields.
